### ISSUE TITLE
[Bug]: "man <command>" shows section 1p rather then section 1

### ISSUE THREAD

**Issue by isrgish** [termux/termux-packages#11342]
### Problem description

When I try to use the man pages I very often get the results from section 1p rather then from section 1. For example `man find` shows the manpage of section 1p. I have tried different solutions that I have found mentioned with no success.

1. export MANSECT=1:1p:8:6:2:3:5:7:4:9:3p
2. Forcing to remake the mandoc.db database by deleting it and running makewhatis on the directory to recreate it.
3. Doing "man -s 1 <command>" (ex. man -s 1 find).

None of these options helped. With one exception if I delete the mandoc.db without remaking it then I do get the manpage from section 1. But then I get a warning that the database is outdated... I don't know if this affects anything (for example that it may not be able to find some of the manpages).

How can I use/have the mandoc.db and still get the correct (order of) manpage sections?

### What steps will reproduce the bug?

Just running `man find`
If the file $PREFIX/share/man/mandic.db then first run `makewhatis $PREFIX/share/man` then run `man find`

### What is the expected behavior?

To get section 1 of the manpages

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=10025
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.dev/apt/termux-main/ stable main
# sources.list.d/science.list
deb https://packages.termux.org/apt/termux-science science stable
# root-repo (sources.list.d/root.list)
deb https://packages.termux.dev/apt/termux-root root stable
# sources.list.d/game.list
deb https://packages.termux.org/apt/termux-games games stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
Updatable packages:
All packages up to date
termux-tools version:
1.23
Android version:
10
Kernel build information:
Linux localhost 4.9.190+ #1 SMP PREEMPT Mon Feb 28 18:54:42 CST 2022 aarch64 Android
Device manufacturer:
LENOVO
Device model:
Lenovo TB-8705F
```

**Comment by sylirre** [comment]
> For example man find shows the manpage of section 1p.

On my device behavior exactly opposite, getting the `1` variant instead of `1p`. Packages `man` and `manpages` are installed. The command is `man find`, `man -s 1 find` also shows the expected (1) page.

Running `makewhatis $PREFIX/share/man` doesn't affect anything.

> But then I get a warning that the database is outdated...

I don't have such warning.

Possibly you installed some package which has "incorrect" manpage file and it cause issues with database.

***

Described steps is not enough to reproduce the issue. I guess we'll need the following:
* Environment variables.
* List of installed packages.

**Comment by isrgish** [comment]
After some testing I found that I will need to give more background to see if someone can help me.

I installed the termux usr folder on an SD card with a symlink to it at `/data/data/com.termux/files/`. I went and copied back the whole folder int `/data/data/com.termux/files/` and tried again to run `man find` now I gt the correct results of section 1.

So in other words having the `usr` folder on sd card doesn't let man work correctly.

What I'm trying to understand. Why do I get these weird results when the folder is on the Sd card? I'm using termux for a long time like this and this is the only issue I have running it like this.
